Resources/Prototypes/Actions/types.yml

```
@@ -69,6 +69,31 @@
     state: old-radio
   event: !type:OpenUplinkImplantEvent
 
+- type: instantAction
+  id: ActivateEmpImplant
+  name: use-emp-implant-action-name
+  description: use-emp-implant-action-description
+  charges: 3
+  useDelay: 10
```

```suggestion
  useDelay: 5
```

10 is awfully high. We could try it as 10 but I think 5 is more than enough. Not required, just a suggestion.

---

I lowered the cooldown to 5 seconds, this should help if multiple people charge you with batons in short intervals.